scripts/info.mk

```diff
@@ -0,0 +1,30 @@
+.PHONY: info info-debug info-swarm
+
+info: ## displays setup information
+	# setup info:
+	@echo ' Detected OS  : $(IS_LINUX)$(IS_OSX)$(IS_WSL)$(IS_WIN)'
```

Comment:
this implicitely means you have IS_LINUX, IS_OSX, IS_WSL and IS_WIN everywhere. is that the case? or maybe you should include these in the file as well.